### PR TITLE
Update to the first stable release 1.0.0

### PR DESCRIPTION
**Pull request by patrickelectric**
Signed-off-by: Patrick José Pereira <patrickelectric@gmail.com>

### CODE REVIEW

File: configuration/index.md [hunk 1]
```diff
@@ -74,58 +74,18 @@ Now that your Companion has an internet connection, you can perform the update t
    ![Version Chooser](version-chooser.png)
 1. If you're already on the latest version, the right side of your Local Version will be blank. If not, you should see a blue **Update** button.
 
-   ![Version Update](version-update.png)
-1. It's also possible to update between different versions (in both directions)
+   <img src="version-update.png" width="650">
+2. It's also possible to update between different versions (When pirate mode is enabled)
```
Comment (ES-Alexander) [inline configuration/index.md:78]: I think repeated `1.` is actually better for this kind of thing - it gets automatically rendered into correct numbering, and makes it possible to switch steps around or add/remove them without needing to re-number everything else.

**Review by ES-Alexander — APPROVED** [review]
Up to you if you want to follow the repeated 1s comment - everything looks fine :-)

We'll need to make sure to update the submodule in `main`

**Comment by ES-Alexander** [comment]
Merging for now because the docs will be linked to in the conference slides

File: configuration/index.md (hunk 1 above)
Comment (patrickelectric) [reply configuration/index.md:78]: That probably was my code editor, sorry for that.

**Pull request merged**